azdev/operations/code_gen.py

```diff
@@ -300,3 +302,61 @@ def _create_package(prefix, repo_path, is_ext, name='test', display_name=None, d
         result = pip_cmd('install -e {}'.format(new_package_path), "Installing `{}{}`...".format(prefix, name))
         if result.error:
             raise result.error  # pylint: disable=raising-bad-type
+
+
+def _get_swagger_readme_file_path(ext_name, swagger_repo):
+    from urllib import request, error
+
+    swagger_readme_file_path = None
+    if swagger_repo == const.GITHUB_SWAGGER_REPO_URL or \
+            (swagger_repo.startswith('https://') and swagger_repo.endswith(const.SWAGGER_REPO_NAME)):
+        swagger_readme_file_path = '{}/blob/master/specification/{}/resource-manager'.format(
+            swagger_repo, ext_name)
+        # validate URL
+        try:
+            request.urlopen(swagger_readme_file_path)
```

Comment:
A question, what if the local copyof readme is out of dated?

Reply:
Then will use the outdated readme file. I think the user should make sure the local readme works or synchronized with the latest remote repo.

Reply:
How about give user a hint? 
Like comparing the digest value between these two files and log that warning out? Just a suggustion, :)

Reply:
User may intend to use the old one, so we cannot help user to use the latest files. And I think it's a little over care for user.